### PR TITLE
update DPP page v0.3.1 schema and @context and with better examples.

### PR DESCRIPTION
**Pull request by onthebreeze**

### CODE REVIEW

**Review by zachzeus — APPROVED** [review]
Looks good

File: website/docs/specification/DigitalProductPassport.md [hunk 1]
```diff
@@ -44,280 +45,250 @@ The digital product passport is designed to meet the following detailed requirem
...
-          "recycled": true,
-          "hazardous": false
-        }
+    "DigitalProductPassport",
+    ],
+"@context":[
+    "https://www.w3.org/ns/credentials/v2",
+    "https://test.uncefact.org/vocabulary/untp/dpp/dpp-context.jsonld"
+    ],
+"id": "acme.com/credentials/2a423366-a0d6-4855-ba65-2e0c926d09b0",
```
Comment (nissimsan) [inline website/docs/specification/DigitalProductPassport.md:90]:
```suggestion
"id": "https://acme.com/credentials/2a423366-a0d6-4855-ba65-2e0c926d09b0",
```

File: website/docs/specification/DigitalProductPassport.md [hunk 2]
```diff
@@ -44,280 +45,250 @@ The digital product passport is designed to meet the following detailed requirem
...
+   "name": "ACME industries",
+   "otherIdentifiers": [{
+      "type": "Entity",
+      "id": "https://abr.business.gov.au/ABN/View?abn=90664869327",
+      "name": "ACME Pty Ltd",
+      "idValue": "90664869327",
+      "idScheme": "abr.business.gov.au",
+      "idSchemeName": "Australian Business Number"}
+    ]},
+"validFrom": "2024.03.15",
```
Comment (nissimsan) [inline website/docs/specification/DigitalProductPassport.md:103]:
```suggestion
"validFrom": "2024-03-15",
```

File: website/docs/specification/DigitalProductPassport.md [hunk 3]
```diff
@@ -44,280 +45,250 @@ The digital product passport is designed to meet the following detailed requirem
...
+"validUntil": "2034.03.15",
```
Comment (nissimsan) [inline website/docs/specification/DigitalProductPassport.md:104]:
```suggestion
"validUntil": "2034-03-15",
```

**Review by nissimsan — COMMENTED** [review]
Nits

File: website/docs/specification/DigitalProductPassport.md [hunk 4]
```diff
@@ -44,280 +45,250 @@ The digital product passport is designed to meet the following detailed requirem
...
+"@context":[
+    "https://www.w3.org/ns/credentials/v2",
+    "https://test.uncefact.org/vocabulary/untp/dpp/dpp-context.jsonld"
+    ],
+"id": "acme.com/credentials/2a423366-a0d6-4855-ba65-2e0c926d09b0",
+"issuer": {
+   "type": "CredentialIssuer",
+   "id": "did:web:identifiers.acme.com:12345",
+   "name": "ACME industries",
+   "otherIdentifiers": [{
```
Comment (nissimsan) [inline website/docs/specification/DigitalProductPassport.md:95]: `alsoKnownAs`

**Review by nissimsan — CHANGES_REQUESTED** [review]

File: website/docs/specification/DigitalProductPassport.md [hunk 5]
```diff
@@ -44,280 +45,250 @@ The digital product passport is designed to meet the following detailed requirem
...
+  "recyclingInformation":{
+     "type": "Link",
+     "linkURL": "acme.com/products/12345/recyclingInfo.html",
+     "linkName": "Recycling instructions",
+     "linkType": "gs1:sustainabilityInfo",
+     "targetType": "application/pdf"
+  } ,
+  "repairInformation": {},
+  "recyclableContent": 50,
+  "recyecledContent": 50
```
Comment (nissimsan) [inline website/docs/specification/DigitalProductPassport.md:287]:
```suggestion
  "recycledContent": 50
```

**Review by nissimsan — APPROVED** [review]

**Pull request merged**